src/main/java/talkabot/TalkaBot.java

```diff
@@ -0,0 +1,166 @@
+package talkabot;
+
+import java.io.IOException;
+import java.time.DateTimeException;
+
+import talkabot.exceptions.InvalidEditException;
+import talkabot.exceptions.InvalidScheduleException;
+import talkabot.exceptions.NoInputException;
+import talkabot.exceptions.UnknownInputException;
+import talkabot.exceptions.TalkaBotException;
+import talkabot.task.Deadline;
+import talkabot.task.Event;
+import talkabot.task.Task;
+import talkabot.task.TaskList;
+import talkabot.task.ToDo;
+
+/**
+ * TalkaBot Class handles the running of Talk-a-Bot
+ * and responds according to user input.
+ */
+public class TalkaBot {
+
+    private Ui ui;
+    private TaskList taskList;
+    private Storage storage;
+
+    /**
+     * Constructor for TalkaBot class.
+     *
+     * @param filePath path of file to be saved in hard drive.
+     */
+    public TalkaBot(String filePath) {
+        this.ui = new Ui();
+        this.storage = new Storage(filePath);
+        try {
+            this.taskList = storage.load();
+        } catch (IOException e) {
+            this.ui.error(e.getMessage());
+        }
+    }
+
+    /**
+     * Runs the Talk-a-Bot, responding based on user input.

```

Comment:
I feel this part can be slightly more specific, like maybe giving an example input and how it works? :)